### ISSUE TITLE
python-dotenv

### ISSUE THREAD

**Issue by cyb3n3tic** [idaholab/Malcolm#216]
Hi, Just in the process of upgrading Malcolm and have completed docker compose pull ok but when running ./scripts/start I am getting the below.

The python-dotenv module is required under Python 3.10.6 (/usr/bin/python3)
System-wide installation varies by platform and Python configuration. Please consult platform-specific documentation for installing Python modules.

Thought I better check and ask if this is a common and easy fix via pip install before doing anything

Thanks in advance

**Comment by mmguero** [comment]
Yeah, `pip3 install python-dotenv` is going to be the best way to do it. Although going forward (next release, I think) I've tweaked it so that it should also work with the .deb package `python3-dotenv` from the latest Ubuntu and Debian even though Ubuntu LTS has got a little bit of an older version in there.

**Comment by cyb3n3tic** [comment]
Good stuff, thanks for confirming
